projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -6,15 +6,40 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
   templateUrl: './chevron.component.html',
 })
 export class SkyChevronComponent {
-  @Output()
-  public directionChange = new EventEmitter<string>();
+  @Input()
+  public ariaControls: string;
 
   @Input()
-  public direction = 'up';
+  public ariaLabel: string;
+
+  @Input()
+  public set direction(value: string) {
+    /* istanbul ignore else */
+    if (value != this._direction) {
+      this._direction = value;
+      /* istanbul ignore else */
+      if (value === 'up') {
+        this.ariaExpanded = true;
+      } else if (this.direction === 'down') {
+        this.ariaExpanded = false;
+      }
+    }
+  }
+
+  public get direction(): string {
+    return this._direction || 'up';
+  }
 
   @Input()
   public disabled = false;
 
+  @Output()
```

Comment:
Do we need a JSDocs comment for this event? I assume so and am guessing it should be something like:

```Fires when the direction of the chevron changes.```